zerver/decorator.py

```diff
@@ -751,8 +755,7 @@ def _wrapped_func_arguments(
                 raise UnauthorizedError(e.msg)
             try:
                 if not skip_rate_limiting:
-                    # Apply rate limiting
-                    rate_limit(request)
+                    rate_limit_user(request, profile, domain="api_by_user")
```

Comment:
Can we add just a quick commit to clean this up  to rename the variable to `user_profile` instead of `profile`, since that's our usual convention?

Reply:
Done.